scripts/install-lansa-scalable.ps1

```diff
@@ -100,6 +100,9 @@ try
 
     # Verify if the ScalableStartHere.htm is present for the Lansa Version
     if (!(Test-Path -Path "$Script:GitRepoPath\Marketplace\LANSA Scalable License\$Cloud\$VersionMajor.$VersionMinor"))  {
+
+        throw 'Cloud/version combination does not exist, default will be considered'
```

Comment:
You must provide sufficient information so that the developer can easily determine what the specific issue is. So, display the Cloud and Version. Must have these values in the output 
```
$Cloud\$VersionMajor.$VersionMinor
```

And 'default will be considered' must be removed. There is no default here. Its just a fatal error.

Also remove the following line of code because it will never be executed.